src/main/java/org/apache/commons/imaging/formats/xpm/XpmImageParser.java

```diff
@@ -10,7 +10,8 @@
  *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
- *  under the License.
+ *
+ * Changed 2015 by Michael Gross, mgmechanics@mgmechanics.de
```

Comment:
This has to be removed from all files. Further more (as stated in CONTRIBUTING.md) it would be good if you fill out an [ICLA](http://www.apache.org/licenses/) if you haven't already.